### PR TITLE
feat: adding podLabels option

### PR DESCRIPTION
**Pull request by frgaudet**
- Adding podLabels support

Signed-off-by: Frédéric Gaudet <frederic.gaudet@blablacar.com>

### CODE REVIEW

File: charts/console/templates/deployment.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ spec:
       {{- end }}
       labels:
         {{- include "console.selectorLabels" . | nindent 8 }}
+        {{- with .Values.podLabels }}{{ toYaml . | nindent 8 }}{{ end }}
```
Comment (joejulian) [inline charts/console/templates/deployment.yaml:23]: No need for `with` here.
```suggestion
        {{ toYaml .Values.podLabels | nindent 8 }}
```

**Review by joejulian — COMMENTED** [review]
Looks good. Just one style change and it's good to merge.

File: charts/console/templates/deployment.yaml (hunk 1 above)
Comment (frgaudet) [reply charts/console/templates/deployment.yaml:23]: Fine with me ;)

File: charts/console/templates/deployment.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ spec:
       {{- end }}
       labels:
         {{- include "console.selectorLabels" . | nindent 8 }}
+        {{ toYaml .Values.podLabels | nindent 8 }}
```
Comment (joejulian) [inline charts/console/templates/deployment.yaml:23]: Ah, my bad, sorry. I was looking at the `with` version thinking _list_. :man_facepalming:

You were right the first time.

Comment (frgaudet) [reply charts/console/templates/deployment.yaml:23]: My bad as well I didn't really pay attention to your suggestion :)

**Review by joejulian — APPROVED** [review]

**Pull request merged**